### PR TITLE
fix deletion of our modules (courtesy @moylop260)

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by moylop260** [comment]
This fixed next iissue of empty folder of runbot-addons:
https://youtu.be/r6-IE87UZQc

:+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by lepistone** [comment]
Ok, that didn't delete the module at all. Hurm.

**Comment by lepistone** [comment]
Amended. Let's see if that's better. (I was deleting `website_twitter` instead of `odoo-extra/website-twitter`).

File: .travis.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ install:
   - travis_install_nightly
   - git clone --depth=1 https://github.com/odoo/odoo-extra ~/odoo-extra
   # odoo-extra has a bunch of v9 modules which aren't compatible, remove them
-  - rm -rf $(ls | grep -v runbot$)
+  - (cd ~/odoo-extra; rm -rf $(ls | grep -v runbot$))
```
Comment (bwrsandman) [inline .travis.yml:36]: Are those outer parenthesis necessary?

**Comment by lepistone** [comment]
This has revealed that `runbot_janitor` tests fail (none were run). Since the failure is now real, I think we can merge this one.

**Comment by pedrobaeza** [comment]
OK, merging. About parenthesis, I think it's not very important if they're redundant because the expression is still very readable

**Comment by bwrsandman** [comment]
@lepistone janitor is not used by anyone, it would be safe to remove it

File: .travis.yml (hunk 1 above)
Comment (lepistone) [inline .travis.yml:36]: they make a subshell, so there is no need to `cd ..` afterwards.

**Comment by lepistone** [comment]
@bwrsandman #54 then. Thanks

**Pull request merged**